app/views/posts/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is list of posts</h1>
\ No newline at end of file
+<div class="posts">
+    <%= image_tag("profile.jpg", :alt => "User icon", width:'100px', height:'100px') %>
+    <% @posts.each do |post| %>
+        <div class="post-list-item">
+            <li><%= link_to post.title, {:action => 'show', :id => post.id}%></li>
+            <p><%= post.text %></p>
+            <h3>Comment <%= post.comments.count%>,  <span>Likes <%= post.likes.count%></span></h3>
+        </div>
+    <div class="comments">
+        <% post.comments.each do |comment| %>
+            <div class="comment">
+                <p><%= comment.text %></p>
+            </div>
+        <% end %>
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires to only render the 5 recent_comments here not all of them. Kindly do that to comply with project requirements.

Reply:
Thanks for pointing it out. Done 😀